public/plugin.ts

```diff
@@ -97,11 +119,33 @@ export class SecurityPlugin
       (config.readonly_mode?.roles || DEFAULT_READONLY_ROLES).includes(role)
     );
 
+    const mountWrapper = async (params: AppMountParameters, redirect: string) => {
+      const { renderApp } = await import('./apps/configuration/configuration-app');
+      const [coreStart, depsStart] = await core.getStartServices();
+
+      // merge OpenSearchDashboards yml configuration
+      includeClusterPermissions(config.clusterPermissions.include);
+      includeIndexPermissions(config.indexPermissions.include);
+
+      excludeFromDisabledTransportCategories(config.disabledTransportCategories.exclude);
+      excludeFromDisabledRestCategories(config.disabledRestCategories.exclude);
+
+      return renderApp(
+        coreStart,
+        depsStart as SecurityPluginStartDependencies,
+        params,
+        config,
+        redirect,
+        deps.dataSourceManagement
+      );
+    };
+
     if (mdsEnabled || apiPermission) {
       core.application.register({
         id: PLUGIN_NAME,
         title: 'Security',
         order: 9050,
+        workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
```

Comment:
```suggestion
        workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
        NavLinkStatus: core.chrome.navGroup.AppNavLinkStatus.getNavGroupEnabled() ? AppNavLinkStatus.hidden : AppNavLinkStatus.visible,
```
Hi @derek-ho , we need to hide the old getting started entry from left navigation if the navGroup is enabled.